packages/web-react/src/components/Header/NavbarClose.tsx

```diff
@@ -21,7 +20,7 @@ const NavbarClose = (props: unknown): JSX.Element => {
       aria-expanded={isExpanded}
       onClick={handleToggle}
     >
-      <span className={classNames(headerIconClass, headerIconCloseClass)} aria-hidden="true" />
+      <Icon name="close" aria-hidden="true" />
```

Comment:
We talked about moving `aria-hidden="true"` into the `Icon` component, could you please fix it?